src/persistence.js

```diff
@@ -19,7 +28,10 @@ export default function _Persistence(mpInstance) {
         );
     };
 
-    this.initializeStorage = function() {
+    this.initializeStorage = function () {
+        if (!this.isEnabled()) {
```

Comment:
This can be moved further down in the method so that we still have some of the initialization occur.